Dockerfile

```diff
@@ -16,6 +16,10 @@ WORKDIR /home/appuser/app
 # Bundle app
 COPY . /home/appuser/app
 
+# Set NODE_ENV; Allow override at build
+ARG NODE_ENV=development
+ENV NODE_ENV=$NODE_ENV
+
```

Comment:
These additions result in errors when using `docker build --build-arg NODE_ENV=production -t app-ui .`:

```
...
Step 10/14 : RUN npm run build
 ---> Running in 79095b77615e

> app-ui@0.0.0 build /home/appuser/app
> run-p bundle

sh: 1: run-p: not found
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! app-ui@0.0.0 build: `run-p bundle`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the app-ui@0.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-01-30T21_02_42_259Z-debug.log
The command '/bin/sh -c npm run build' returned a non-zero code: 1
```